src/storage-controller/Cargo.toml

```diff
@@ -30,14 +30,23 @@ mz-repr = { path = "../repr" }
 mz-service = { path = "../service" }
 mz-storage-client = { path = "../storage-client" }
 mz-storage-types = { path = "../storage-types" }
+mz-timely-util = { path = "../timely-util" }
 mz-tls-util = { path = "../tls-util" }
 once_cell = { version = "1.16.0" }
 proptest = { version = "1.0.0", default-features = false, features = ["std"] }
 prost = { version = "0.11.3", features = ["no-recursion-limit"] }
 serde = { version = "1.0.152", features = ["derive"] }
 serde_json = { version = "1.0.89" }
-timely = { version = "0.12.0", default-features = false, features = ["bincode"] }
-tokio = { version = "1.24.2", features = ["fs", "rt", "sync", "test-util", "time"] }
+timely = { version = "0.12.0", default-features = false, features = [
```

Comment:
No idea why my formatter for `.toml` files is different from everyone else's will spend a few minutes trying to solve that before mergin this.